geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ConcurrentParallelGatewaySenderEventProcessor.java

```diff
@@ -309,9 +309,6 @@ protected void waitForResumption() throws InterruptedException {
     super.waitForResumption();
   }
 
-  /**
-   * Test only methods for verification purpose.
-   */
   public List<ParallelGatewaySenderEventProcessor> getProcessors() {
```

Comment:
It's ok to remove the javadocs, but you should add the annotation `@VisibleForTesting`.

Any access modifier (private/package-private/protected/public) that is made less restrictive than `private` simply to allow a test to access it should have `@VisibleForTesting` added to it.

Reply:
I remove this javadoc as we don't use this method only for testing.